RFCXXX-Native-Json-Adapters.md

```diff
@@ -0,0 +1,258 @@
+> into the pipeline immediately following the original native application.
+
+A reasonable approach may be to specify the name of the application, then provide a scriptblock to determine the value of a given parameter.
+This would be somewhat more complicated than our current `PSDefaultParameterValues` but the complexity is needed to evaluate the proper values to provide.
+
+The proposed architecture makes it fairly trivial to add an adapter which only calls `ConvertFrom-Json`.
+The combination of this feature, and the follow-on work to automatically adding parameters to native applications, and our formatting system could change this:
+
+```powershell
+PS> Docker image ls --format='{{json .}}' | ConvertFrom-Json | Where-Object {$_.Repository -match "microsoft"}| Format-Table Id,Repository,Tag,Size,CreatedAt
+
+ID           Repository                             Tag                     Size   CreatedAt
+--           ----------                             ---                     ----   ---------
+4e9617ada198 mcr.microsoft.com/powershell           preview-7.3-mariner-2.0 235MB  2022-09-01 11:48:17 -0700 PDT
+4bcdf53ee67b mcr.microsoft.com/powershell           latest                  339MB  2022-09-01 11:34:40 -0700 PDT
+97a21961f76d mcr.microsoft.com/mssql/server         2022-latest             1.6GB  2022-08-18 16:59:49 -0700 PDT
+02214dbfb621 mcr.microsoft.com/powershell/test-deps ubi-8.4                 640MB  2022-08-11 11:35:47 -0700 PDT
+e3afdc6d8e5c mcr.microsoft.com/mssql/server         2019-latest             1.47GB 2022-07-25 01:45:18 -0700 PDT
+```
+
+to this:
+
+```powershell
+PS> docker image ls --format='{{json .}}'| Where-Object Repository -match "microsoft"
+
+ID           Repository                             Tag                     Size   CreatedAt
+--           ----------                             ---                     ----   ---------
+4e9617ada198 mcr.microsoft.com/powershell           preview-7.3-mariner-2.0 235MB  2022-09-01 11:48:17 -0700 PDT
+4bcdf53ee67b mcr.microsoft.com/powershell           latest                  339MB  2022-09-01 11:34:40 -0700 PDT
+97a21961f76d mcr.microsoft.com/mssql/server         2022-latest             1.6GB  2022-08-18 16:59:49 -0700 PDT
+02214dbfb621 mcr.microsoft.com/powershell/test-deps ubi-8.4                 640MB  2022-08-11 11:35:47 -0700 PDT
+e3afdc6d8e5c mcr.microsoft.com/mssql/server         2019-latest             1.47GB 2022-07-25 01:45:18 -0700 PDT
+```
+
+#### Automatic Conversion of Tabular Text
+
+If the data is _known_ to be tabular it may be possible to _automatically_ convert that tabular data to
+structured data via means of a generalized converter.
+This would require some sort of registration/configuration to designate the native application and call the
+converter to automatically construct objects from tabular data could be called.
+This would enable the user to set it once and later sessions would be able to use it.
+
+#### PSDefaultNativeParameterValues
```

Comment:
It's not clear to me if this section is actually proposing work to be done or just calling out a potential feature and problems with it.  It seems like it's the latter which I would then recommend moving to `Alternate Considerations` which implies it's not in scope